stripecardscan/src/main/java/com/stripe/android/stripecardscan/scanui/SimpleScanActivity.kt

```diff
@@ -340,6 +340,7 @@ internal abstract class SimpleScanActivity<ScanFlowParameters> : ScanActivity()
     }
 
     protected open fun setupPrivacyLinkTextUi() {
```

Comment:
Can we make this not an open function? that would also make it harder to be modified